lib/broken_record/console_aggregator.rb

```diff
@@ -18,7 +19,7 @@ def report_results(klass)
 
       if all_errors.any?
         puts "#{all_errors.length} errors were found while running validations for #{klass}\n"
-        puts "Invalid ids: #{all_errors.keys.inspect}"
```

Comment:
this line here was failing because the contract was broken. it was no longer a hash